docs/react-v9/contributing/common-dev-snags.md

```diff
@@ -1,4 +1,16 @@
-// TODO #30642 A doc explaining common snags and trip hazards.
+### Cleaning your repo
 
-Assignee - Mason
-Reviewer - Sean
+Every once in a while you'll get build errors in node modules unrelated to your work. This usually happens when changes get checked into packages not related to your repo.
```

Comment:
> This usually happens when changes get checked into packages not related to your repo.

Maybe just me, but I'm not sure I understand what this means. What defines a package not related to your repo?

Reply:
good catch - needs re-wording.